scripts/requirements-ci.txt

```diff
@@ -79,6 +79,7 @@ pytz==2020.5
 pyusb==1.1.1
 PyYAML==5.4.1
 recommonmark==0.6.0
+regex==2020.11.13
```

Comment:
`regex` should also be added here:
https://github.com/nrfconnect/sdk-nrf/blob/master/scripts/requirements-build.txt

Users are not expected to install pip packages from `requirements-ci.txt` but they should still be able to run tests.
Alternatively under:
https://github.com/nrfconnect/sdk-nrf/blob/master/scripts/requirements-extra.txt

Reply:
Done